### ISSUE TITLE
problem with using GLF  format as input 

### ISSUE THREAD

**Issue by cityCoder** [ANGSD/angsd#25]
Hi,
I use ANGSD's genotype likelihood function to dump results in Binary GLF files. And I tried to perform analyze using ANGSD with such GLF file as input. For example,
angsd -glf binary.glf.gz  -fold 1 -nInd 2 -anc ucsc.hg19.fasta  -doSaf 1 -out test -fai ucsc.hg19.fasta.fai,
however, I got lots of errors, such as:

Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171349 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171399 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171449 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171499 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171549 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171599 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171649 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171699 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171749 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171799 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171849 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171899 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171949 ref_len=16571
Trying to access fasta efter end of chromsome+200:chrM/chrM pos=2171974 ref_len=16571
        -> Done reading data waiting for calculations to finish
        -> Done waiting for threads

        -> npools:0 unfreed tnodes before clean:0
        -> Output filenames:
                ->"test.arg"
                ->"test.saf.gz"
                ->"test.saf.pos.gz"
                ->"test.saf.idx"
        -> Thu Dec 17 14:39:35 2015
        -> Arguments and parameters for all analysis are located in .arg file
        [ALL done] cpu-time used =  2.43 sec
        [ALL done] walltime used =  3.00 sec

I would like to know is there any  reasons for such problem.

Thanks very much

**Comment by ANGSD** [comment]
The pval cutoff depends on which analysis you want to perform. You can consider using the minmaf filter which is simpler to interpret.

Which -doglf option did you use? There are 4 possible ones:
Filedumping:
	-doGlf	0
	1: binary glf (10 log likes)	.glf.gz
	2: beagle likelihood file	.beagle.gz
	3: binary 3 times likelihood	.glf.gz
	4: text version (10 log likes)	.glf.gz

the 1) does not contain information about positions. This has mostly been an internal format. But let me know if you want the positions to be written in a separate file. 
Since the  format doesnt have any information of the position, the reading defaults to calling the positions 'dummy1 1','dummy1 2' etc. So you would need to make a dummy fai file just containing some chr name and then some value, that will have to be atleast the number of sites in your file.

Best

**Comment by cityCoder** [comment]
Before your response, I tried several format, finally, I selected beagle.gz format. But I am not sure whether this format works for me or not. 
As declared both on the wiki and the paper, ANGSD supports various input format. I have several bam files, but they are very large, it might be a bottle neck when processing those data, especially when I want to partially redo the analysis. So I came up with the idea that save GL files, and use those GL files as input for most of the analyze.
Thanks a lot
